staff/tomas-sancheznavarro/js-arrays/find.js

```diff
@@ -0,0 +1,14 @@
+function find(arr, func) {
+
+    if (typeof arr === 'undefined') throw Error('undefined is not an array');
+    if (typeof func === 'undefined') throw Error('undefined is not a function');
+
```

Comment:
Maybe the Error message could be rephrased so that the user has more info (like: arr + 'is not an array' or func + 'is not a function')

Reply:
Correction applied! Thanks!